src/dune_rules/file_binding.mli

```diff
@@ -21,6 +21,12 @@ module Unexpanded : sig
   val make : src:Loc.t * string -> dst:Loc.t * string -> t
 
   val expand :
+       t
+    -> dir:Path.Build.t
+    -> f:(String_with_vars.t -> string Memo.Build.t)
+    -> Expanded.t Memo.Build.t
+
+  val expand_pure :
```

Comment:
We generally use the term "static" in other places. Though, I expect that will eventually get rid of them all once we have pushed the monad enough.

Reply:
Indeed, fixed.